src/Host.cpp

```diff
@@ -2700,7 +2700,7 @@ std::pair<bool, QString> Host::openWindow(const QString& name, bool loadLayout,
     //Dont create Userwindow if there is a Label with the same name already. It breaks the UserWindow
     auto pL = mpConsole->mLabelMap.value(name);
     if (pL) {
-        return {false, QStringLiteral("label with the name \"%1\" exists already. userwindow name has to be unique").arg(name)};
+        return {false, QStringLiteral("label with the name \"%1\" exists already: userwindow name has to be unique").arg(name)};
```

Comment:
No idea how : connects the 2 sentences. I would expect information on where or how the name exists already. 
I would connect the sentences with "but" or leave them separated or drop the second completely.